### ISSUE TITLE
Settings based switch clock billability [feature request (or bug report)]

### ISSUE THREAD

**Issue by cegento** [peerigon/clockodo#116]
Hello, 
currently the clockodo rest API supports a small but cool feature in which you can have billability based on your projects settings, by omitting the `billable` key (in the api request) from the parameters. (https://www.clockodo.com/en/api/clock/#c15189-headline).

If i do the same thing with the javascript clockodo api, i get the following error: `Unhandled error: Error: Missing required parameter "billable" at Proxy.checkRequired`

The request is currently done like in this example: 
```js
    const parameters = {
        customersId: customersId,
        projectsId: projectsId,
        servicesId: servicesId,
        text: text,
        usersId: usersId
    }
    if (unbillable) {
        parameters.billable = 0;
    }
    await clockodoApi.startClock(parameters);
```

If now the variable `unbillable` is false [to get the projects settings billability], the field billable will be undefined which causes the above mentioned error. I also don't know whether i am doing a mistake, or whether the feature is not implemented yet.

Thanks! :)

**Comment by jhnns** [comment]
Thanks for bringing it up. That makes totally sense :)
I'll look into it

**Comment by sprankhub** [comment]
Does https://github.com/peerigon/clockodo/commit/da085faebfa9159d553824552ee604b6094237c3 fix that issue? :thinking: Thanks, @jhnns!

**Comment by sprankhub** [comment]
https://github.com/peerigon/clockodo/pull/117 sounds like it. Great!

**Comment by jhnns** [comment]
Yes, this has been fixed with #117 :)
